packages/apps/fortune/exchange-oracle/server/package.json

```diff
@@ -13,6 +13,13 @@
     "start:prod": "node dist/main",
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
     "test": "jest",
+    "migration:create": "typeorm-ts-node-commonjs migration:create",
```

Comment:
We should add build command in migration scripts, otherwise non build changes won't be considered